perses/distributed/feptasks.py

```diff
@@ -199,14 +200,24 @@ def run_protocol(thermodynamic_state, sampler_state, ne_mc_move, topology, n_ite
         An MDtraj topology for the system to generate trajectories
     n_iterations : int
         The number of times to apply the specified MCMove
+    atom_indices_to_save : list of int, default None
+        list of indices to save (when excluding waters, for instance). If None, all indices are saved.
 
     Returns
     -------
     trajectory : mdtraj.Trajectory
         Trajectory containing n_iterations frames
 
     """
-    n_atoms = topology.n_atoms
+    #get the atom indices we need to subset the topology and positions
+    if atom_indices_to_save is None:
+        atom_indices = range(topology.n_atoms)
```

Comment:
Do you want a `range` object here (python3 behavior)? The user would normally provide a list. Perhaps it doesn't make a difference, but just want to make sure to convert to list, or face a potential bug.

Reply:
Thanks, good call